### ISSUE TITLE
Deployment Target lower than supported os version

### ISSUE THREAD

**Issue by Koushik-Ogma** [SDWebImage/SDWebImage#3121]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | ios
 Platform Version        | 13.0
 SDWebImage Version      | 5.9.4
 Integration Method      | cocoapods
 Xcode Version           | Xcode 12
 Repro rate              | all the time (100%)

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

The library works completely fine **but** there are build time warnings. The compiler says:
> The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.0.99.

#### In Package.swift
```Swift
platforms: [
        .macOS(.v10_10),
        .iOS(.v8),
        .tvOS(.v9),
        .watchOS(.v2)
    ]
```
Please update the iOS version so that we can get rid of the warnings.

Thanks!

**Comment by dreampiggy** [comment]
See #3054 .
Available in 6.0, not 5.0. Because deployment target version should effect user, we can not release a 5.x version which bump to iOS 9.0+.

**Comment by Koushik-Ogma** [comment]
I understand. Thanks a lot for your reply.
Feel free to close the issue.
